Practical/CRUD_Application/script.js

```diff
@@ -0,0 +1,159 @@
+//Validate form inputs before submitting data
+function validateForm() {
+  var pname = document.getElementById("pname").value;
+  var pid = document.getElementById("pid").value;
+  var pi = document.getElementById("pi").value;
+  var pd = document.getElementById("pd").value;
+  var pp = document.getElementById("pp").value;
+
+  if (pname == "") {
+    alert("Producrt Name is required");
+    return false;
+  }
+
+  if (pid == "") {
+    alert("Producrt Identification must be specified");
+    return false;
+  }
+
+  if (pi == "") {

```

Comment:
No alert is appearing 
also code gets broken gives error in console when no image is uploaded. It should provide alert but not error in console.